CHANGELOG-table.md

```diff
@@ -0,0 +1 @@
+- I created a table about the donor's race.
```

Comment:
```suggestion
- Created a table about the donor's race.
```
Style: In the context of the changelog, we usually just have lots of sentence fragments. The "I" doesn't matter.